docker/Dockerfile

```diff
@@ -64,37 +49,83 @@ FROM base as builder
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
+# Add PostgreSQL's third party repository to get the latest versions
+RUN curl -o /usr/share/keyrings/postgresql.asc -s https://www.postgresql.org/media/keys/ACCC4CF8.asc
+RUN echo "deb [signed-by=/usr/share/keyrings/postgresql.asc] http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" | tee /etc/apt/sources.list.d/pgdg.list
+
+# Add apt-fast repo
+RUN curl -o /usr/share/keyrings/apt-fast.asc -s https://keyserver.ubuntu.com/pks/lookup?op=get\&search=0xA2166B8DE8BDC3367D1901C11EE2FF37CA8DA16B
+RUN echo "deb [signed-by=/usr/share/keyrings/apt-fast.asc] http://ppa.launchpad.net/apt-fast/stable/ubuntu jammy main" | tee /etc/apt/sources.list.d/apt-fast.list
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends apt-fast -y \
+    && rm -rf /var/lib/apt/lists/*
+
 # Install Rust (specific version) and other build dependencies
 RUN apt-fast update && apt-fast install -y --no-install-recommends \
     build-essential \
+    checkinstall \
```

Comment:
I don't think we need to install checkinstall anymore (or is it used behind the scenes by deb?)